spec/features/idv/analytics_spec.rb

```diff
@@ -185,18 +187,39 @@
     before do
       allow(IdentityConfig.store).to receive(:in_person_proofing_enabled).and_return(true)
 
+      start_idv_from_sp
```

Comment:
by default the sp is `oidc` by changing it to `saml` this method then calls `visit_idp_from_saml_sp_with_ial2` which will have the cta button present. It will however not have the link to the sp. Error will instead be `expect(page).to have_content(t('in_person_proofing.body.barcode.return_to_partner_html'))`

This feels like progress though, will explore creating a 3rd sp example or including url for sp. For the latter option I think I need to better look at this method: `visit_idp_from_saml_sp_with_ial2` in `idv_helper.rb`